docker-compose.yml

```diff
@@ -23,6 +25,21 @@ services:
       - ./storage/mysql:/var/lib/mysql
     env_file:
       - .env.mysql
+
   browser:
     hostname: browser
-    image: selenium/standalone-chrome
\ No newline at end of file
+    image: selenium/standalone-chrome
+
+  phpmyadmin:
+    build: phpmyadmin
+    environment:
```

Comment:
This works great on my machine. The only thing I'd suggest is maybe declaring these environment variables in some sort of .env file in a similar way to how the mysql variables are declared

Reply:
Done